config/locales/activerecord.en.yml

```diff
@@ -133,6 +133,7 @@ en:
             # Marking &email_address for reference later.
             value: &email_addres_validations
               invalid: “%{value}” is not a valid email address
+              missing_mx_records: “%{value}” is not a valid email address
```

Comment:
actually for the Polish translation looks like you can just copy whatever is in there for the `invalid` translation right above this line

Reply:
Done.